### ISSUE TITLE
Loss and Learning Rate Schedule

### ISSUE THREAD

**Issue by LucasSloan** [AlexeyAB/darknet#5696]
I'm trying to take the advice in https://github.com/AlexeyAB/darknet#how-to-improve-object-detection about how to improve model performance by using higher resolution images.  To be precise, I've run trained 2 models on the same dataset, one at 416x416, and another at 512x512.

The training run at 416x416 leveled out at a loss of ~3, and a mAP of 34 after ~20k batches:

![chart](https://user-images.githubusercontent.com/102313/81123952-1fc8d100-8ee9-11ea-9e9a-c276032347f1.png)

The training run at 512x512 leveled out out a loss of ~4, but had a small increase in loss at 12800 batches (when I stepped down the learning rate) and a huge increase in loss at 14400 batches (when I stepped down the learning rate again):

![chart](https://user-images.githubusercontent.com/102313/82596583-5dba2c00-9b5c-11ea-8866-5fbd0f003d1b.png)

Is it expected that the loss will be higher when training at a higher resolution?  Do those jumps in loss when I step down the loss rate imply anything about how many steps I should train for/what I should set the learning rate at?  I've noticed that the [yolov4 config](https://github.com/AlexeyAB/darknet/blob/master/cfg/yolov4.cfg) trains at 608x608 resolution and a learning rate of 0.00261, both of which are higher than the values in the [yolov4-custom config](https://github.com/AlexeyAB/darknet/blob/master/cfg/yolov4-custom.cfg).
